packages/@uppy/core/src/index.js

```diff
@@ -1031,6 +1037,11 @@ class Uppy {
    * @return {string} ID of this upload.
    */
   _createUpload (fileIDs) {
+    const { allowNewUpload, currentUploads } = this.getState()
+    if (!allowNewUpload) {
+      throw new Error('Cannot create a new upload: already uploading.')
+    }
```

Comment:
Seems like a “hard” error, will this be shown in the UI?

Reply:
Checked, this is inside the promise chain created in upload(), and will be shown in the UI